data/brands/shop/supermarket.json

```
@@ -9747,6 +9747,21 @@
         "name:en": "Coop Sapporo",
         "shop": "supermarket"
       }
+    },
+    {
+      "displayName": "Рода",
+      "locationSet": {"include": ["rs"]},
+      "tags": {
+        "brand": "Рода",
+        "brand:en": "Roda",
```

```suggestion
        "brand:en": "Roda",
        "brand:sr": "Рода",
```
If you add a `*:en` but original language isn't English, you may need duplicate original language's tag, see https://wiki.openstreetmap.org/wiki/Multilingual_names#Repeating_name_with_language_specific_tag